api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -33,15 +33,16 @@
 	
 	// Constructor
 	public CohortMembership() {
+		startDate = null;
```

Comment:
I think this line can be omitted as startDate would be null if not initialized.

Reply:
Yes. You are right. I will consider removing this line. I will check if similar attributes are initialized with null in constructor and i will take this decision. Thanks.